biometric-login/test/configs/wdio.android.sauce.emu.conf.js

```diff
@@ -96,7 +117,16 @@ config.capabilities = [
         locale: 'en',
         // Group builds by build name
         build: buildName,
+        name: 'Android 7.1 Biometric Login',
     },
 ];
 
+const appiumVersion = process.env.npm_config_appium_version
```

Comment:
Nice!

Reply:
we can add more things like, an array of platforms to test with?? :D

Reply:
Looks great, but maybe keep it how it is now to keep is _simple_